test/cfa/ntp_conf_test.rb

```diff
@@ -80,7 +80,18 @@ def ntp_conf(file)
         ntp.records << record
         ntp.save
         expect(file.content.lines).to include("server 3.pool.ntp.org# test comment\n")
+      end
 
+      it "can write multi lines comments from autoyast profiles" do
+        record = CFA::NtpConf::ServerRecord.new
+        record.value = "3.pool.ntp.org"
+        record.comment = "# test comment\n#test comment 2\n#test comment3"
+        ntp.records << record
+        ntp.save
+        expect(file.content.lines).to include("server 3.pool.ntp.org\n")
+        expect(file.content.lines).to include("## test comment\n")
```

Comment:
well, it really depends on how lense is written. There are lenses which have "#" matched and stored, so it is part of final string, and some that just require it, but do not store it, so it is not part of final string. It is a bit inconsistent there. And even more challenging in files that allow both "#" and ";" as comment starter.